git_machete/docs.py

```diff
@@ -662,6 +662,24 @@
 
           <b>--no-detect-squash-merges</b>         Only consider "strict" (fast-forward or 2-parent) merges, rather than rebase/squash merges,
                                             when detecting if a branch is merged into its upstream (parent).
+
+        To make it easier to select branch name from the `status's` output on certain terminals (E.g. Alacritty), you can add additional
```

Comment:
I'd add some section here, similar to `Options`... like `Config keys`?

Reply:
Sure, should make it more readable